backend/src/test/java/ch/puzzle/okr/service/persistence/AlignmentPersistenceServiceIT.java

```diff
@@ -130,6 +134,85 @@ void findByObjectiveAlignmentIdShouldReturnListOfAlignments() {
         assertAlignment(alignments.get(0));
     }
 
+    @Test
+    void recreateEntityShouldUpdateAlignmentNoTypeChange() {
```

Comment:
Test hat 2 Probleme:
- ist zu lang und kompliziert
- ein Tests sollte nur eine Sache testen. Hier testest du verschiedene Dinge

Wenn du das nur zusammenhängend testen kannst dann "wenigstens":
- aufsplitten setup und asserts in private Helper Methoden

PS: das gleiche gilt für die nächste Test Methode